vermin/formats/default_format.py

```diff
@@ -1,17 +1,25 @@
 from .format import Format
 from ..utility import version_strings, sort_line_column
-from ..printing import vprint
+from ..printing import vprint, sgr
 
 class DefaultFormat(Format):
-  def __init__(self):
-    super().__init__("default")
+  def __init__(self, colored=False):
+    super().__init__("colored" if colored else "default")
```

Comment:
Really good idea with the ANSI colored format!
We just need tests of that as well.
That is the last part before I will merge this PR. 🎉

Reply:
And adding it to README, too.